modules/bloxone_infra_host_aws/variables.tf

```diff
@@ -47,7 +47,7 @@ variable "services" {
   description = "The services to provision on the BloxOne Host. The services must be a map of valid service type with values of \"start\" or \"stop\". Valid service types are \"dhcp\" and \"dns\"."
```

Comment:
Please update the description as well in each module.

Reply:
done